spine_json_lib/test/test_spine_animation_editor.py

```diff
@@ -210,3 +211,18 @@ def test_erase_skins(self):
             imgs_refs_expected = json.load(f)
 
         assert DeepDiff(animation_editor.images_references, imgs_refs_expected) == {}
+
+    def test_missing_required_attr(self):
```

Comment:
I would change the name for something more self-explanatory, there are a lot of of required attributes